test/extension/exponent/exponentHelper.test.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import { ExponentHelper } from "../../../src/extension/exponent/exponentHelper";
+import * as path from "path";
+import * as assert from "assert";
+import * as sinon from "sinon";
+import * as Q from "q";
+import { FileSystem } from "../../../src/common/node/fileSystem";
+
+suite("exponentHelper", function() {
+    const RESOURCES_ROOT = path.resolve(__dirname, "../../resources/exponentHelper");
+
+    async function checkIsExpoApp(packageJson: any, expected: boolean) {
+        let fs = new FileSystem();
+        sinon.stub(fs, "readFile", () => Q.resolve(JSON.stringify(packageJson)));
+        const expoHelper = new ExponentHelper(RESOURCES_ROOT, "", fs);
```

Comment:
Is the second empty arg have no influence to ExponentHelper object?

Reply:
No.